### PR TITLE
Update to ZH normalizer regex

### PR DESCRIPTION
**Pull request by shiki-tm**
I added 3 syllable separating symbols that I think should be added to the regex. I hope I did it right. The symbols and where they can be found sometimes:

    '     most dictionaries use this like cedict
    ’      兩岸詞典 and 白水社中国語辞典
    -      wenlin abc sometimes

### CODE REVIEW

**Comment by Casheeew** [comment]
@shiki-tm tests are failing

**Comment by shiki-tm** [comment]
I'm sorry I'm not sure what's wrong

File: test/language/chinese-reading-normalizer.test.js [hunk 1]
```diff
@@ -25,6 +25,9 @@ const tests = [
     ['wán:zhěng', 'wánzhěng'],
     ['fān・yì', 'fānyì'],
     ['fān//yì', 'fānyì'],
+    ['fān’yì', 'fānyì'],
+    ['fān'yì', 'fānyì'],
```
Comment (djahandarie) [inline test/language/chinese-reading-normalizer.test.js:29]: You need to escape the `'` here, otherwise the string gets broken. That's why CI is failing

**Comment by Casheeew** [comment]
@shiki-tm sorry, unfortunately I've been a bit busy this week and couldn't look closely at the problem. I'll look into it this afternoon

**Comment by shiki-tm** [comment]
@Casheeew oh its good, thanks. i dont know why i was trying to do this, i dont even use pinyin but i think i was trying to do this for a new version of cedict marv might release in the future with the single quote. but after thinking about it, the single quote for separating pinyin syllables is there for a good reason...the single quote shouldnt be hidden i think now. ive also pretty much perfected the pinyin to zhuyin conversion so maybe this kind of update should only be considered if someone who uses pinyin finds other legitimate stuff to hide from the pinyin readings. im sorry this was a waste of time

**Comment by Casheeew** [comment]
> but after thinking about it, the single quote for separating pinyin syllables is there for a good reason...the single quote shouldnt be hidden i think now.

The normalizer doesn't remove the separators, it just tells that two entries with and without the separator are the same (so we can sort and display them better in the popup). The highest entry, if it has separators in the pinyin, would still retain them so it shouldn't be a concern. (the way it is implemented isn't ideal but I was considering for the exact reason you had)

**Comment by Casheeew** [comment]
@shiki-tm https://github.com/shiki-tm/yomitan/pull/1

**Comment by shiki-tm** [comment]
Thank you. Please let me know if I have to do anything else

**Review by Casheeew — APPROVED** [review]

**Comment by Casheeew** [comment]
> Thank you. Please let me know if I have to do anything else

Nothing needed. Thanks a lot!

**Pull request merged**